iOS/Objective C/IronSourceDemoApp/DemoViewController.m

```diff
@@ -77,25 +87,33 @@ - (void)setupIronSourceSdk {
     self.rewardedVideoDelegate = [[DemoRewardedVideoAdDelegate alloc] initWithDelegate:self];
     [IronSource setLevelPlayRewardedVideoDelegate:self.rewardedVideoDelegate];
     
-    self.interstitialDelegate = [[DemoInterstitialAdDelegate alloc] initWithDelegate:self];
-    [IronSource setLevelPlayInterstitialDelegate:self.interstitialDelegate];
-    
-    self.bannerDelegate = [[DemoBannerAdDelegate alloc] initWithDelegate:self];
-    [IronSource setLevelPlayBannerDelegate:self.bannerDelegate];
     
     self.impressionDataDelegate = [[DemoImpressionDataDelegate alloc] init];
     [IronSource addImpressionDataDelegate:self.impressionDataDelegate];
-    
-    self.initializationDelegate = [[DemoInitializationDelegate alloc] initWithDelegate:self];
-    
+        
     // After setting the delegates you can go ahead and initialize the SDK. 
     // Once the iniitaliztion callback is return you can start loading your ads
-    [self logMethodName:@"initWithAppKey:delegate:"];
-    [IronSource initWithAppKey:kAppKey 
-                      delegate:self.initializationDelegate];
     
-    // To initialize specific ad units:
-    // [IronSource initWithAppKey:kAppKey adUnits:@[IS_REWARDED_VIDEO, IS_INTERSTITIAL, IS_BANNER] delegate:self.initializationDelegate];
+    // Init the SDK when implementing the Multiple Ad Units Interstitial and Banner API, and Rewarded using legacy APIs
+    LPMInitRequestBuilder *requestBuilder = [[LPMInitRequestBuilder alloc] initWithAppKey:kAppKey];
+    [requestBuilder withLegacyAdFormats:@[IS_REWARDED_VIDEO]];
+
+    LPMInitRequest *initRequest = [requestBuilder build];
+    
+    [self logMethodName:[NSString stringWithFormat:@"init ironSource SDK with appKey: %@", kAppKey]];
```

Comment:
I would move this line before the requestBuilder, and then the initRequest parameter will be close the [LevelPlay initWithRequest] without spaces.